scripts/views.py

```diff
@@ -737,6 +712,53 @@ def vote_for_script(request, pk: int, version: str) -> None:
     update_user_related_script(models.Vote, request.user, script_version)
     return redirect(request.POST["next"])
 
+def map_similar_scripts(data):
+    return {
+        'value': data[1],
+        'name': data[0].script.name,
+        'scriptPK': data[0].script.pk,
+    }
+
+# Seperate call to calculate similar scripts so we can lazy load it
+def get_similar_scripts(request, pk: int, version: str) -> JsonResponse:
+    if request.method != "GET":
+        raise Http404()
+
+    current_script = models.ScriptVersion.objects.filter(
```

Comment:
**nitpick** I generally prefer .get over .filter for single elements. I'll make a separate PR for that since that's the only feedback I have.

Reply:
Ah yeah makes sense, I don't touch much python